crypto/objects/objects.txt

```diff
@@ -1524,7 +1524,7 @@ secg-scheme 14 3 : dhSinglePass-cofactorDH-sha512kdf-scheme
 1 3 6 1 4 1 311 60 2 1 3	: jurisdictionC		: jurisdictionCountryName
 
 # SCRYPT algorithm
-1 3 6 1 4 1 11591 4 11		: id-scrypt
+1 3 6 1 4 1 11591 4 11		: scrypt            : id-scrypt
```

Comment:
Typo in comment which I thought I'd fixed. I meant add ": scrypt" at the end. That should keep SN_id_scrypt unchanged (which we should do for compatibility reasons) and add a new LN_id_scrypt.